Croogo/Console/Templates/croogo/views/admin_view.ctp

```diff
@@ -68,7 +68,16 @@ foreach ($fields as $field) {
 	}
 	if ($isKey !== true) {
 		echo "\t\t<dt><?php echo __d('croogo', '" . Inflector::humanize($field) . "'); ?></dt>\n";
-		echo "\t\t<dd>\n\t\t\t<?php echo h(\${$singularVar}['{$modelClass}']['{$field}']); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+		switch ($schema[$field]['type']) {
+			case 'datetime':
+				echo "\t\t<dd>\n\t\t\t<?php echo \$this->Time->format(\${$singularVar}['{$modelClass}']['{$field}'], '%Y-%m-%d %H:%M:%S', __d('croogo', 'Invalid datetime')); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+				break;
+			case 'boolean':
+				echo "\t\t<dd>\n\t\t\t<?php echo \$this->Html->status(\${$singularVar}['{$modelClass}']['{$field}']); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+				break;
+			default:
+				echo "\t\t<dd>\n\t\t\t<?php echo h(\${$singularVar}['{$modelClass}']['{$field}']); ?>\n\t\t\t&nbsp;\n\t\t</dd>\n";
+		}
 	}
 }
 ?>
```

Comment:
This is the only change for admin_view but there should be more. I am not utilizing admin_view templates at the moment and thus I did not notice. Generated content looks like this
```php
///...
		<dt><?php echo __d('croogo', 'Name'); ?></dt>
		<dd>
			<?php echo h($tariff['Tariff']['name']); ?>
			&nbsp;
		</dd>
//...
```
E.g. the translation domain is still wrong for those files.